package.json

```diff
@@ -93,6 +93,7 @@
     "scratch-l10n": "2.0.20180108132626",
     "scratch-paint": "0.2.0-prerelease.20180410152401",
     "scratch-render": "0.1.0-prerelease.1523453612",
+    "scratch-svg-renderer": "latest",
```

Comment:
in GUI we are pinning all dependencies to stabilize the build process, so you should use the result of `npm info scratch-svg-renderer version` instead of latest once you merge that bitmap import PR?